syli9526/ArchitectureStudy/app/src/main/java/com/example/architecturestudy/network/UpbitService.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.architecturestudy.network
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class UpbitService {
```

Comment:
인스턴스를 `getInstance()` 로만 가져올 수 있어야 합니다
```suggestion
class UpbitService private constructor() {
```

Reply:
[f4119d8](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/184/commits/f4119d8a5a7c493ec51320edf71309ffec7a735a)
수정하였습니다.